src/main/java/httpserver/HTTPHandler.java

```diff
@@ -22,232 +22,187 @@
  * @see MessageHandler
  */
 public abstract class HTTPHandler {
```

Comment:
Should we rename our classes to follow standard convention? Meaning they'd all change from `HTTPClass` to `HttpClass`. This would be `HttpHandler`.